### PR TITLE
Topic Model Analytics View support

### PR DESCRIPTION
**Pull request by hideaki**
# Description
Topic Model Analytics View support.

# Checklist
Make sure you have performed the following items before submitting this pull request.
If not, please describe the reason.  

- [x] Add test cases for this fix/enhancement
- [ ] Pass devtools::check()
- [ ] Pass devtools::test()
- [ ] Test installing from github
- [x] Tested with Exploratory

### CODE REVIEW

File: R/textanal.R [hunk 1]
```diff
@@ -379,24 +401,146 @@ exp_text_cluster <- function(df, text,
...
+    }
+    else {
+      res <- res %>% dplyr::slice_max(value, n=num_top_words, with_ties=FALSE) # Set hard limit even with ties.
+    }
+
+    res <- res %>% dplyr::ungroup() %>%
       dplyr::rename(cluster = document)
+
+    # Extract count of words in each cluster from dfm_cluster.
+    res <- res %>% dplyr::nest_by(cluster) %>% ungroup() %>%
```
Comment (kei51e) [inline R/textanal.R:435]: Add 'dplyr::'

File: R/textanal.R [hunk 2]
```diff
@@ -379,24 +401,146 @@ exp_text_cluster <- function(df, text,
...
+      dplyr::mutate(data=purrr::map2(data, cluster, function(y, clstr){y %>%
+                                     dplyr::mutate(count=as.numeric(x$dfm_cluster[clstr,token_id]))})) %>% tidyr::unnest(data)
```
Comment (kei51e) [inline R/textanal.R:437]: Improve indentation.

File: R/textanal.R [hunk 3]
```diff
@@ -379,24 +401,146 @@ exp_text_cluster <- function(df, text,
...
     res
```
Comment (kei51e) [inline R/textanal.R:438]: Remove res.

File: R/textanal.R [hunk 4]
```diff
@@ -379,24 +401,146 @@ exp_text_cluster <- function(df, text,
...
+                                        remove_punct = remove_punct, remove_numbers = remove_numbers,
+                                        stopwords_lang = stopwords_lang, stopwords = stopwords,
+                                        hiragana_word_length_to_remove = hiragana_word_length_to_remove,
+                                        compound_tokens = compound_tokens)
+
+    # convert tokens to dfm object
+    dfm_res <- tokens %>% quanteda::dfm()
+
+    lda_model <- seededlda::textmodel_lda(dfm_res, k = num_topics)
+    docs_topics <- lda_model$theta
```
Comment (kei51e) [inline R/textanal.R:487]: Add comment about docs_topics.

File: R/textanal.R [hunk 5]
```diff
@@ -379,24 +401,146 @@ exp_text_cluster <- function(df, text,
...
+
+    docs_sample_index <- if (nrow(docs_topics) > mds_sample_size) {
```
Comment (kei51e) [inline R/textanal.R:489]: Add a comment.

File: R/textanal.R (hunk 1 above)
Comment (hideaki) [reply R/textanal.R:435]: Done.

File: R/textanal.R (hunk 2 above)
Comment (hideaki) [reply R/textanal.R:437]: Done.

File: R/textanal.R (hunk 3 above)
Comment (hideaki) [reply R/textanal.R:438]: Done.

File: R/textanal.R (hunk 4 above)
Comment (hideaki) [reply R/textanal.R:487]: Done.

File: R/textanal.R (hunk 5 above)
Comment (hideaki) [reply R/textanal.R:489]: Done.

**Review by kei51e — APPROVED** [review]

**Pull request merged**